packages/beyondinsight_password_safe/changelog.yml

```
@@ -1,3 +1,11 @@
+- version: "0.7.0"
+  changes:
+    - description: For the managedsystem data stream, fixed session management to automatically re-authenticate when sessions expire.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/15007
+    - description: For the managedsystem data stream, renamed fields to match proper snake_case naming - `ipaddress` to `ip_address` and `isarelease_duration` to `isa_release_duration`.
```

```suggestion
    - description: For the `managedsystem` data stream, fixed session management to automatically re-authenticate when sessions expire.
      type: bugfix
      link: https://github.com/elastic/integrations/pull/15007
    - description: For the `managedsystem` data stream, renamed fields to match proper snake_case naming - `ipaddress` to `ip_address` and `isarelease_duration` to `isa_release_duration`.
```
for consistency with the entry below.

---

5324835